pages/api/main-metrics.js

```diff
@@ -215,6 +217,89 @@ const fetchMechFees = async () => {
   }
 };
 
+const PREDICT_AGENT_IDS = [13, 14, 25, 9, 26, 29, 37, 36, 33, 44, 46, 45];
```

Comment:
can all these calculations be moved to pages/api/predict-metrics.js please? otherwise we request all main metrics on predict page which don't belong to it